daemon/k8s/init.go

```diff
@@ -168,3 +169,26 @@ func restoreRouterHostIPs(n *nodeTypes.Node, log logrus.FieldLogger) {
 		}).Info("Restored router IPs from node information")
 	}
 }
+
+// restoreHealthIPs restores (sets) the health IPs found from the
+// Kubernetes resource.
+func restoreHealthIPs(n *nodeTypes.Node, log logrus.FieldLogger) {
+	if !option.Config.EnableHealthIPRestore {
+		return
+	}
+
+	health4 := n.IPv4HealthIP
+	health6 := n.IPv6HealthIP
+	if health4 != nil {
+		node.SetEndpointHealthIPv4(health4)
+	}
+	if health6 != nil {
+		node.SetEndpointHealthIPv6(health6)
+	}
+	if health4 != nil || health6 != nil {
```

Comment:
nit: Should we just log individually in the condition above? 
nit: Also might be best to declare the vars were they are used.

Reply:
Done